parser/parser.y

```diff
@@ -14241,6 +14242,26 @@ PlanReplayerStmt:
 			x.Limit = $10.(*ast.Limit)
 		}
 
+		$$ = x
+	}
+|	"PLAN" "REPLAYER" "DUMP" "EXPLAIN" stringLit
+	{
+		x := &ast.PlanReplayerStmt{
+			Stmt:    nil,
+			Analyze: false,
+			Load:    false,
+			File:    $5,
+		}
+		$$ = x
+	}
+|	"PLAN" "REPLAYER" "DUMP" "EXPLAIN" "ANALYZE" stringLit
+	{
+		x := &ast.PlanReplayerStmt{
+			Stmt:    nil,
+			Analyze: true,
+			Load:    false,
+			File:    $6,
```

Comment:
In `PLAN REPLAYER LOAD 'dump_file'`, we set `PlanReplayerStmt.File` to `dump_file`. Maybe we can add comment to explain different usages of `PlanReplayerStmt.File` between `PLAN REPLAYER DUMP` and `PLAN REPLAYER LOAD` or we can use different fields for different usages.